Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,47 @@
-﻿namespace Logger;
+﻿# nullable enable
+﻿using System;
+
+namespace Logger;
 
 public static class BaseLoggerMixins
```

Comment:
```suggestion
public static class BaseLoggerMixins : BaseLogger
```
BaseLoggerMixins needs to extend BaseLogger

Reply:
extension methods not derive